modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+= TinyMCE 6.2
+:navtitle: TinyMCE 6.2
+:description: Release notes for TinyMCE 6.2
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.2 was released for {enterpriseversion} and {cloudname} on Monday, September 26^th^, 2022. It includes {productname} 6.2, additional changes to Premium plugins, and several new Premium plugins. These release notes provide an overview of the changes for {productname} 6.2:
+
+* xref:new-premium-plugins[New Premium plugins]
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
```

Comment:
The list and the left navbar say that the `Improvements` section should be before the `Additions` section. Although, that doesn't look to be the case below

Reply:
(discussion): decision was to move all the notes themselves in the content so that additions are after improvements